README.md

```diff
@@ -20,13 +20,25 @@ And then execute:
 
 API_KEY needed.
 
-TODO: Describe how to get a YouTube API key.
+Include a .env file in the root of the project directory and add your YouTube API Key in plain text aka without "", '', or backticks
```

Comment:
I think what we tried to say by "how to get a YouTube API key" was actually how you get the key from Google console.